src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -487,5 +487,8 @@ public void PressDialogButton(string dialogAutomationName, string buttonAutomati
                 throw new InvalidOperationException($"The element {buttonAutomationName} does not have an InvokePattern. Please make sure that it is the correct control");
             }
         }
+
+        public void UndoLastGlobalAction()
+            => GetDTE().ExecuteCommand("Edit.UndoLastGlobalAction");
```

Comment:
I thought the VS Editor owned the concept of Undo, which is why I put it there. Thoughts?